modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-feature-search/ddm-template.ftl

```diff
@@ -0,0 +1,399 @@
+<style>
+	a.link-container:hover, a.link-container.hover {
```

Comment:
These are all pretty generic styles. That may be okay, but just want to make sure we really want to effect every class on any page this is loaded on. Typically it is better to scope all of the styles in a template to apply to just this template. This can be done by adding a class to the top wrapper and then prepending every css selector with that class (or nesting the css)